lib/transaction.js

```diff
@@ -86,7 +86,7 @@ class Transaction {
                     line.toString('binary').replace(/\r\n$/, '\n'));
             }
         }
-        else if (this.header_pos && this.parse_body) {
+        else if (this.parse_body) {
```

Comment:
Are you sure about this change?  This change makes the test fixture *different* than Haraka/transaction.js.